en/features/details/fabricated-transactions.md

```diff
@@ -0,0 +1,50 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+layout: post
+lang: en
+columns: 1
+id: details-fabricated-transactions
+title: Details about fabricated transactions
+breadcrumbs:
+  - features
+  - verification
+---
+> Mallory creates a transaction giving Alice 1,000 bitcoins, so Alice
+> gives Mallory some cash. Later Alice discovers the transaction Mallory
+> created was fake.
+
+- **Wallet service**{:.fgred} users depend on the information reported
+  by the service, so they can easily be fooled into accepting fabricated
+  transactions.
+
+- **Light wallet**{:.fgred} users depend on full nodes and
+  miners to verify transactions for them. It costs nothing for
+  dishonest full nodes to send unconfirmed fabricated transactions to a
+  light wallet. Getting one or more confirmations of those fabricated
+  transactions is also possible with help from a dishonest miner.
```

Comment:
I think the original is more useful.  "Confirmed" to some people might only mean one confirmation; we want to convey that it's possible even to get six confirmations.

Reply:
Good point, I agree.  

Should it be "Getting one or more confirmations on those fabricated transactions..." or "Getting one or more confirmations for those fabricated transactions..."  instead of "confirmations of"?